.github/workflows/ci.yml

```diff
@@ -65,16 +64,46 @@ jobs:
         run: |
           pip3 install git+https://github.com/executablebooks/sphinx-book-theme.git
           echo GITHUB_SHA_SHORT=${GITHUB_SHA::8} >> $GITHUB_ENV
-          mkdir ${DEST_FOLDER}
+          mkdir html
       - name: Build documentation
         run: |
-          export BUILD_TYPE=${BUILD_TYPE} && sphinx-build doc/main ${DEST_FOLDER}
+          export BUILD_TYPE=${BUILD_TYPE} && sphinx-build doc/main html
+          tar -czvf html.tar.gz html/
       - name: Save docs
         uses: actions/upload-artifact@v2.2.1
         with:
           name: oneTBB-html-docs-${{ env.GITHUB_SHA_SHORT }}
           path: html
 
+  pages:
+    if: ${{ github.ref == 'refs/heads/master' }}
+    runs-on: ubuntu-latest
+    needs: [documentation]
+    steps:
+      - name: Checkout gh-pages
+        uses: actions/checkout@v2
+        with:
+          ref: gh-pages
```

Comment:
Make sure you create orphan branch `gh-pages` in the repo before merging the PR.

Reply:
Created